paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -46,8 +47,9 @@ fun detectEnvironment(): Environment {
   val configLines = File("build/intermediates/paparazzi/resources.txt").readLines()
   val packageName = configLines[0]
   val resDir = configLines[1]
+  val targetSdkVersion = configLines[2].toInt()
```

Comment:
Continuing from above, even ignoring custom SDKs and ignoring old SDKs from Google that contained Google APIs which had a different format, in the case of only official Android SDKs you will not always get an integer value. When the preview SDK for the next version of Android is made available it will be named `android-R`.

Reply:
> Based on this, I think you're better off pulling the _actual_ target SDK version from the Gradle model which is always going to be an integer value.

how to grab this value?